snapcraft/plugins/_python/_pip.py

```diff
@@ -124,6 +126,10 @@ def _setup(self):
                 # Now that we have our own pip, reset the python home
                 self._python_home = real_python_home
 
+    def is_setup(self):
```

Comment:
Given the docstring, this should probably be renamed to `is_pip_setup`

Reply:
And given what it does, I would just make `_is_pip_installed` public and use that instead

Reply:
And if so, this is probably missing a test, please double check.

Reply:
I agree with promoting `_is_pip_installed` to public, but I'd rather call it `is_setup()` so we can have the `setup`/`is_setup` logical pair.

Reply:
Done.